api/src/org/apache/cloudstack/api/command/admin/vlan/UpdateVlanIpRangeCmd.java

```diff
@@ -0,0 +1,115 @@
+package org.apache.cloudstack.api.command.admin.vlan;
+
+import org.apache.cloudstack.api.APICommand;
+import org.apache.cloudstack.api.ApiConstants;
+import org.apache.cloudstack.api.ApiErrorCode;
+import org.apache.cloudstack.api.BaseAsyncCmd;
+import org.apache.cloudstack.api.Parameter;
+import org.apache.cloudstack.api.ServerApiException;
+import org.apache.cloudstack.api.response.VlanIpRangeResponse;
+import org.apache.log4j.Logger;
+
+import com.cloud.dc.Vlan;
+import com.cloud.event.EventTypes;
+import com.cloud.exception.ConcurrentOperationException;
+import com.cloud.exception.ResourceAllocationException;
+import com.cloud.exception.ResourceUnavailableException;
+import com.cloud.user.Account;
+
+
+@APICommand(name = "updateVlanIpRange", description = "Updates a VLAN IP range.", responseObject =
+        VlanIpRangeResponse.class, since = "4.9.0",
```

Comment:
Shouldn't this be 4.13?

Reply:
based on #3111 I thought we can push it back to 4.9, should I switch it to 4.13?

Reply:
4.9 is no longer supported nor maintained and there will be no new release.

As this is a new feature it should go into 4.13

Reply:
Alright, changed to 4.13